tests/FakeItEasy.Analyzer.Tests/UnusedCallSpecificationAnalyzerTests.cs

```diff
@@ -97,17 +98,115 @@ interface IFoo { int Bar(); }
                 test,
                 new DiagnosticResult
                 {
-                    Id = UnusedCallSpecificationAnalyzer.DiagnosticId,
+                    Id = DiagnosticDefinitions.UnusedCallSpecification.Id,
                     Message =
-                        "Unused call specification for 'foo.Bar()'; did you forget to configure or assert the call?",
+                        "Unused call specification for 'A.CallTo(() => foo.Bar())'; did you forget to configure or assert the call?",

```

Comment:
A last-minute comment: I think the "for" should be struck here (well, where it's generated). `A.CallTo(() => foo.Bar())` is the call specification, it's not _for_ it.